lib/manager/composer/types.ts

```diff
@@ -8,6 +8,11 @@ export interface Repo {
 }
 export interface ComposerConfig {
   type?: string;
+  config?: {
+    platform?: {
+      php?: string;
```

Comment:
It looks like renovate should also be able to suggest updated to this config? At least security updates.

Reply:
I agree that would be very useful, I'm unsure how that would be implemented though.

Reply:
No problem, out of scope for this PR, but we should open a follow up issue

Reply:
I'll make a note to do so, thanks for the suggestion!